src/main/java/seedu/address/model/tag/ModuleTag.java

```diff
@@ -15,15 +15,47 @@ public class ModuleTag extends Tag implements Comparable<ModuleTag> {
             "NUS Modules should have 2 - 3 letter prefix, followed by 4 digits and optional 1 - 3 alphabets";
     public static final String VALIDATION_REGEX = "[A-Z]{2,4}[0-9]{4}[A-Z]{0,3}";
 
+    private String day;
+    private String startTime;
+    private String endTime;
+    private String location;
+    private String latitude;
+    private String longitude;
+
+
     /**
      * Constructs a {@code ModuleTag}.
      *
+     * @param tagName A string for tag names.
+     */
+    public ModuleTag(String[] tagName) {
```

Comment:
Separate Constructor to take in String arrays.

Reply:
You mean "Overloading constructor"?

Reply:
yea you are right

Reply:
Good choice.

Reply:
Also, you could try doing the varargs, as it achieves the same thing but is more malleable. So like String… tagNames.

Reply:
Oh and the tagName should really be tagNames.